CMakeLists.txt

```diff
@@ -366,6 +366,9 @@ if(NOT QSCINTILLA_FOUND)
     link_directories("${CMAKE_CURRENT_BINARY_DIR}/${QSCINTILLA_DIR}")
 endif()
 
+message("find me")
```

Comment:
Found it!  Do I win a prize? 😁

Reply:
Haha, just some trial and error with Travis